EmotiBitFirmwareInstaller/src/ofApp.cpp

```diff
@@ -331,7 +340,16 @@ void ofApp::keyPressed(int key){
 
 //--------------------------------------------------------------
 void ofApp::keyReleased(int key){
-
+	if (((char)key) == 'L')
```

Comment:
The user can still press `L` in any other state in the state machine and open a "windows explorer/ finder" dialog.
This will, however, only change the `_fwFilePath` variable, and nothing else.

Should we add a `if` check to detect the button press if only in `State::DISPLAY_INSTRUCTION` ? @produceconsumerobot

Reply:
[patched](https://github.com/EmotiBit/ofxEmotiBit/pull/169/commits/505303383265378cefe253d82ba517d9cb1348ae)